### PR TITLE
Corrections in the docs

### LINKED ISSUE #910: Errors in the docs
Found several errors in the docs:

## Step

There are still several mentions of `tippy` in the docs.

---

arrow | Object | Whether to display the arrow for the tooltip or not.
-- | -- | --

- `Object` should be `Boolean`
- Maybe add the defaults (`true` in this case)?

---

```js
const new Step(tour, {
  attachTo: { element: '.some .selector-path', on: 'left' },
  ...moreOptions
})'
```

- First line should start with `const step = new Step(...`
- Replace trailing `'` with `;` on last line
- Same goes for the other examples

### PR DESCRIPTION
**Pull request by sn3p**
closes #910

### CODE REVIEW

File: docs-src/tutorials/02-usage.md [hunk 1]
```diff
@@ -157,15 +157,15 @@ created.
 - `title`: The step's title. It becomes an `h3` at the top of the step.
 - `attachTo`: What element the step should be attached to on the page.
 It should be an object with the properties `element` and `on`, where `element` is an element selector string
-or a DOM element and `on` is the optional direction to place the Tippy tooltip.
-              
+or a DOM element and `on` is the optional direction to place the step element.
```
Comment (rwwagner90) [inline docs-src/tutorials/02-usage.md:160]:
```suggestion
or a DOM element and `on` is the optional direction to place the Popper tooltip.
```

File: src/js/step.js [hunk 2]
```diff
@@ -133,7 +133,7 @@ export class Step extends Evented {
   }
 
   /**
-   * Remove the step, delete the step's element, and destroy the tippy instance for the step
+   * Remove the step and delete the step's element
```
Comment (rwwagner90) [inline src/js/step.js:136]:
```suggestion
   * Remove the step, delete the step's element, and destroy the Popper instance for the step.
```

File: docs-src/tutorials/02-usage.md (hunk 1 above)
Comment (sn3p) [reply docs-src/tutorials/02-usage.md:160]: Popper is not really a "tooltip", but can do 👍

Comment (rwwagner90) [reply docs-src/tutorials/02-usage.md]: We're creating tooltips with Popper though

**Comment by rwwagner90** [comment]
Thanks for the PR!

File: docs-src/tutorials/02-usage.md (hunk 1 above)
Comment (sn3p) [reply docs-src/tutorials/02-usage.md]: True :)

**Pull request merged**